### DISCUSSION TITLE
Make it harder to disable 2FA

### DISCUSSION THREAD

**Discussion by tomkidd** [community/community#65940] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I just went through the deal to add 2FA, downloaded recovery codes, set up an app, etc. 

Then I accidentally clicked the green "Enabled" button in my profile (that indicates 2FA is enabled) and it INSTANTLY disabled all 2FA and I had to do it again. 

Being able to disable 2FA is good, but I think this green "Enabled" button needs an "Are you sure?" style dialog prompt to stop people like me who clicked on it by accident. Just a suggestion.

**Comment by zacharysierakowski** [comment]
Hi there @tomkidd 👋 

Thanks for your feedback! This was something already in our backlog that our team had been wanting to get to, but hadn't prioritized yet.

As of yesterday, we deployed an "are you sure?" style dialog prompt implementation just as you suggested 🙂 You should be able to see that on github.com today.

![image](https://github.com/community/community/assets/22333813/45dd456f-4683-4360-991a-cf228976eda1)

**Comment by Mornieur** [comment]
response #3
